File: src/main/java/it/contrader/dto/DataSetDTO.java
package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(generator=ObjectIdGenerators.PropertyGenerator.class, property="id")
public class DataSetDTO {

	
	private Long id;

	private Long utente;
	
	private Long categoria;
	private String categoriaN;
	
	private Long unitaMisura;
	private String unitaMisuraN;
	
	private String valore;
	
	private String commento;

}

File: src/dto/DataSetDTO.ts
export class DataSetDTO{

    private id : number;

	private utente : number;
	
	private categoria : number;
	private categoriaN : string;
	
	private unitaMisura : number;
	private unitaMisuraN : string;
	
	private valore: string;
	
    private commento : string;
    
    constructor(id : number, utente : number, categoria : number, categoriaN : string, unitaMisura : number, unitaMisuraN : string, valore: string, commento : string){
        this.id = id;
        this.utente = utente;
        this.categoria = categoria;
        this.categoriaN = categoriaN;
        this.unitaMisura = unitaMisura;
        this.unitaMisuraN = unitaMisuraN;
        this.valore = valore;
        this.commento = commento;
    }
}